src/pages/ThorChainLP/components/ReusableLpStatus/TransactionRow.tsx

```diff
@@ -122,7 +122,7 @@ export const TransactionRow: React.FC<TransactionRowProps> = ({
 
   const { data: fromAddress } = useQuery({
     ...reactQueries.common.thorchainFromAddress({
-      accountId: fromAccountId,
+      accountId: fromAccountId!,
```

Comment:
Don't slap me but `fromAccountId` is a part of the `enabled` prop so it will be defined if the query run

Also, I wan't to avoid to refactor this query to use `skipToken` because this part is not so related to the base PR